### PR TITLE
Update test.py

### PR DESCRIPTION
**Pull request by dilipchauhan14**
Added footer.

### CODE REVIEW

File: test.py [hunk 1]
```diff
@@ -89,3 +89,6 @@ def get_merged_by(merged_by):
         return None
     else:
         return merged_by[LOGIN]
+
```
Comment (dilipchauhan14) [inline test.py:92]: Following Line Follows footer.

File: test.py [hunk 2]
```diff
@@ -89,3 +89,6 @@ def get_merged_by(merged_by):
         return None
```
Comment (dilipchauhan14) [inline test.py:89]: Usual above it.

File: test.py [hunk 3]
```diff
@@ -89,3 +89,6 @@ def get_merged_by(merged_by):
...
     else:
```
Comment (dilipchauhan14) [inline test.py:90]: Usual 2 Lines below it.

**Review by dilipchauhan14 — COMMENTED** [review]
Marked Usuals.

**Review by sammarth-kapse — APPROVED** [review]

**Pull request merged**